### ISSUE TITLE
[Accessibility] On viewport page width of 320 CSS pixels output pane content is cut off.

### ISSUE THREAD

**Issue by Krituhar** [microsoft/vscode#227626]
- VS Code Version: 1.88.0
- OS Version: MacOS Sonoma (v14.6.1)

**Issue**: The output page content is requiring horizontal scrolling to view when the page has a viewport width of 320 CSS pixels. Accessibility users with low vision who need to resize text will be prevented from accessing this content. 

**[Recommendation]**
Ensure pages reflow without requiring two-dimensional scrolling without loss of content or functionality. When the page has a viewport width of 320 CSS pixels, all content and functionality must still be available without requiring horizontal scrolling. The content shouldn't get cut off with smaller screen size. 

Moving content to an accessible show/hide control, such as a hamburger menu, is acceptable.

Screenshot of issue:
<img width="448" alt="Screenshot 2024-09-04 at 1 55 11 PM" src="https://github.com/user-attachments/assets/43f85318-b50d-4e4e-911e-5fb30ade668e">

**Comment by meganrogge** [comment]
Not sure i'm following - in your screenshot, you're showing the output view with dev tools open. Are you suggesting that the output view should grow vertically when the width is narrow so a user doesn't have to scroll horizontally?

**Comment by meganrogge** [comment]
I've just tested this by making the window very small, something I imagine low vision users would not do very often. Zooming in works well and VS Code is resizing in a responsive way as the window gets smaller. 

One thing we could do is to have a minimum window size, but that would be unnecessarily restrictive for users. Thus, closing this.
